plugins/UsersManager/lang/en.json

```diff
@@ -1,5 +1,10 @@
 {
     "UsersManager": {
+        "2FA": "2FA",
+        "UsesTwoFactorAuthentication": "Uses two-factor authentication",
+        "TwoFactorAuthentication": "Two-factor authentication",
+        "ResetTwoFactorAuthentication": "Reset two-factor authentication",
+        "ResetTwoFactorAuthenticationInfo": "If the user cannot log in anymore because of lost recovery codes and the authentication device you can reset two-factor authentication so the user can log in again.",
```

Comment:
Translation tweak: `"If the user can no longer log in due to lost recovery codes or a lost authentication device, you can reset two-factor authentication for the user so they can log in again."`

Reply:
👍  will update